ocamltest/ocaml_actions.ml

```diff
@@ -582,6 +582,38 @@ let ocamlobjinfo =
          Result.skip_with_reason "ocamlobjinfo not available", env
     )
 
+let ocamltest_action log env =
```

Comment:
Note for reviewers: to review this, just compare to the `mklib` stuff below -- on the github interface this requires expanding the diff a bit.